docs/resources/aws_ec2_images.md

```diff
@@ -0,0 +1,82 @@
+---
+title: About the aws_ec2_images Resource
+platform: aws
+---
+
+# aws\_ec2\_images
+
+Use the `aws_ec2_images` InSpec audit resource to test properties of some or all AWS EC2 images.
+
+## Syntax
+
+ Ensure you have exactly 3 EC2 Images
```

Comment:
Perhaps saying `images available to you` or `images you have launch permissions` would be more definitive.

Reply:
Updated in recent commit.